### PR TITLE
Atox120 scripts

### PR DESCRIPTION
**Pull request by atox120**

### CODE REVIEW

File: gym_homer/envs/homer_env.py [hunk 1]
```diff
@@ -229,39 +248,42 @@ def _update_battery(self, solar, loads, max_d, max_c, action
         # Calculate Grid State, Update Battery state
```
Comment (bcormier1) [inline gym_homer/envs/homer_env.py:248]: I think we should remove the "discrete" code path.  We should model the "discrete max" case as the fractional with an interval of 1.  
ex: 
```interval = 1
action space = 2*1 +1 = 3   -> [0,1,2]

// in step() the action is then modeled as 
self.updated_action = (action - self.action_intervals) / self.action_intervals
updated_action = [(0-1) /1 ,(1-1) /1, (2-1) /1] = [-1,0,1]

// functionally [max_discharge, no action, max_charge]
```
Maintaining two code paths is asking for bugs (we've already seen a couple)

**Review by bcormier1 — CHANGES_REQUESTED** [review]
I found a bug in the arithmetic for the action space and updated the branch. This allows us to remove the "discrete" case because it can be done implicitly with the fractional (easier hyper-parameter sweeping too)

File: gym_homer/envs/homer_env.py (hunk 1 above)
Comment (atox120) [reply gym_homer/envs/homer_env.py:247]: agreed.

File: gym_homer/envs/archive/homer_env_01.py [hunk 2]
```diff
@@ -0,0 +1,372 @@
+"""
+Date: 17th November 2022
```
Comment (bcormier1) [inline gym_homer/envs/archive/homer_env_01.py:2]: This is just for backup?

Comment (atox120) [reply gym_homer/envs/archive/homer_env_01.py:2]: yep

**Pull request merged**